10.9-libcxx/stable/main/finkinfo/devel/meson-py.info

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; tab-width: 4 -*-
+Info4: <<
+Package: meson-py%type_pkg[python]
+Version: 0.53.2
+Revision: 1
+Maintainer: Leigh Smith <leigh@leighsmith.com>
+Type: python (3.6 3.7 3.8)
+# setuptools needed because meson uses pkg_resources package.
+Depends: <<
+	python%type_pkg[python],
+	setuptools-tng-py%type_pkg[python]
+<<
+Source: https://files.pythonhosted.org/packages/49/d7/38a71de6adaa7a25a72587e69429ce78e137f4d545c7516341aa081f43e7/meson-%v.tar.gz
+Source-Checksum: SHA256(3e8f830f33184397c2eb0b651ec502adb63decb28978bdc84b3558d71284c21f)
+DocFiles: COPYING README.md
+CompileScript: <<
+ %p/bin/python%type_raw[python] setup.py build
+<<
+# Rename the installed files to match the python variant.
+InstallScript: <<
+ %p/bin/python%type_raw[python] setup.py install --root %d
+  mv %i/bin/meson %i/bin/meson-py%type_pkg[python]
+  mv %i/share/man/man1/meson.1 %i/share/man/man1/meson-py%type_pkg[python].1
+  mv %i/share/polkit-1/actions/com.mesonbuild.install.policy %i/share/polkit-1/actions/com.mesonbuild.install.policy-py%type_pkg[python]
```

Comment:
```suggestion
  perl -pi -e 's|(/usr)(/bin/python)(3)|%p${2}%type_raw[python]|;' %i/share/polkit-1/actions/com.mesonbuild.install.policy
  perl -pi -e 's|(/usr)(/bin/meson)|%p${2}-py%type_pkg[python]|;' %i/share/polkit-1/actions/com.mesonbuild.install.policy
  mv %i/share/polkit-1/actions/com.mesonbuild.install.policy %i/share/polkit-1/actions/com.mesonbuild.install.policy-py%type_pkg[python]
```
Again pointing to the versioned executables as installed (admittedly I have no idea exactly what the `com.mesonbuild.install.policy` file is doing, so this might be wrong...)